cernopendata/modules/fixtures/data/records/jade-logbooks.json

```diff
@@ -1,7 +1,7 @@
 [
   {
     "abstract": {
-      "description": "This is JADE logbook 01 from 1970. FIXME add more description."
+      "description": "This is JADE logbook 01 from 10.03.1979 to 09.08.1979 with runs 1 to 1337 described on 189 pages. These logbooks are a handwritten collection of incidents in the format of a timeline and a list of properties for most of the Runs done with the PETRA accelerator. This specific one contains a number of descriptions of computer errors, near overheating of the record tapes and the first signs of three tracks."
```

Comment:
I'll change this to March 10th if that's OK for you. Otherwise 09.08 may be confusing, depending on a country people can read it as DD.MM or MM.DD.